examples/kubernetes/vtgate-service.yaml

```diff
@@ -7,7 +7,10 @@ metadata:
     app: vitess
 spec:
   ports:
-    - port: 15001
+    - name: web+bsonrpc
```

Comment:
Since it's possible to refer to ports by name from other configs in k8s, I suggest we name this something we won't have to change when bsonrpc is disabled (given that it's already deprecated). In the gRPC-only future, this port will still be needed solely for the web interface, until gRPC is able to multiplex web+RPC on a single port.